.github/workflows/dotnet.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Build
       run: dotnet build --no-restore
     - name: Test
-      run: dotnet test --no-build --verbosity normal -l "console;verbosity=detailed" --collect:"XPlat Code Coverage" --logger "trx;LogFileName=test-results.trx" --results-directory ./coverage
+      run: dotnet test --filter 'BOLT8|BOLT9|BOLT10' --no-build --verbosity normal -l "console;verbosity=detailed" --collect:"XPlat Code Coverage" --logger "trx;LogFileName=test-results.trx" --results-directory ./coverage
```

Comment:
Instead of using an allow list, we should just disallow the execution of the Docker tests like:

`--filter 'FullyQualifiedName!~Docker'`